starter-code/src/viking.js

```diff
@@ -1,11 +1,108 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  receiveDamage(damage) {
+    let vikingHealth = (this.health -= damage);
+    if (vikingHealth > 0) {
+      return `${this.name} has received ${damage} points of damage`;
+    } else if (vikingHealth <= 0) {
```

Comment:
no need to do an `else if`, can directly return your string because it is clear that this will only return if the vikingHealth > 0 is not true on line 24